.github/policies/scheduledSearches.yml

```diff
@@ -208,52 +207,52 @@ configuration:
                 >   - The "Status: No Recent Activity :zzz:" label must be removed.
                 >   - If applicable, the "Status: Long Term :hourglass_flowing_sand:" or the "Needs: Module Owner :mega:" label must be added.
 
-      - description: 'ITA05A - Close issues that have been marked as requiring author feedback but have not had any activity for 3 days, unless it''s been marked with the "Status long term" label.'
-        frequencies:
-          - hourly:
-              hour: 3
-        filters:
-          - isIssue
-          - isOpen
-          - hasLabel:
-              label: "Needs: Author Feedback :ear:"
-          - hasLabel:
-              label: "Status: No Recent Activity :zzz:"
-          - isNotLabeledWith:
-              label: "Needs: Module Owner :mega:"
-          - noActivitySince:
-              days: 3
-        actions:
-          - addReply:
-              reply: |
-                > [!WARNING]
-                > @${issueAuthor}, this issue will now be closed, as it has been marked as requiring author feedback but has not had any activity for **7 days**.
+      # - description: 'ITA05A - Close issues that have been marked as requiring author feedback but have not had any activity for 3 days, unless it''s been marked with the "Status long term" label.'
```

Comment:
Leftover?

Reply:
Yes, just as you wrote (and as noted on the sister-PR), I intentionally didn't remove this, but left it there, as we might need it - or another form of it at a later point. If you check the related documentation updates, you'll see that I also captured it as disabled, instead of completely removing it.

Please resolve. Thanks.

Reply:
Approved the PR but will leave @AlexanderSehr to resolve this

Reply:
ok, as long as you have an eye on it 😉